extensions/chromium/preferences_schema.json

```diff
@@ -159,6 +159,10 @@
       "type": "boolean",
       "default": false
     },
+    "ignoreDestinationZoom": {
```

Comment:
Should this be exposed in the UI (in that case there should probably be a title/description added)?

Reply:
We have seen quite a few questions about this particular behavior, and personally I also always found it more confusing than helpful, so I think it's good to expose this in the UI with a description saying that the option overrides the behavior that PDF files can change the zoom level.

Reply:
I'm always finding these sort of descriptions/explanations somewhat difficult to write (in such a way that they really benefit users), but at least the patch is now updated to address this point (and it's always possible to improve the wording later on).